frontend/src/test/scala/bloop/ForkerSpec.scala

```diff
@@ -116,6 +127,20 @@ class ForkerSpec {
     }
   }
 
+  @Test
+  def canHandleLongClasspaths(): Unit = TestUtil.withinWorkspace { tmp =>
+    val longCp = Array.fill(JvmProcessForker.classpathCharLimit / 10) {
+      AbsolutePath(Files.createTempFile("forkerspec-temp", ".jar"))
+    }
+
+    run(tmp, Array("foo", "bar", "baz"), longCp) {
```

Comment:
Could we double-check in this test that we're indeed using the MANIFEST to run the code (maybe a debug log that we detect via `logger`)? Otherwise we're not sure if this test is actually checking the process char limit logic.